tests/test_fmi3.py

```diff
@@ -0,0 +1,13 @@
+# TODO: Remove/rename this file
+import sys
+from pathlib import Path
+
+this_dir = Path(__file__).parent.absolute()
+sys.path.append(str(this_dir))
+
+from utils import setup_reference_fmus
+
+setup_reference_fmus()
+expected_fmu = Path(this_dir) / 'files' / 'reference_fmus' / '3.0' / 'VanDerPol.fmu'
+if not expected_fmu.exists():
+    raise Exception(f"Test setup failed, FMU {expected_fmu} does not exist!")
```

Comment:
This is just a temporary sanity check to verify it works. Once we start adding tests that involve these FMUs I think we should just remove above code.

Reply:
I think it is OK to keep, to have a simple sanity check that the download script doesn't break. 

The actual testing should be moved into a test function, so the result actually shows up with pytest.

Reply:
I have moved it now, it was also required by pytest once I added the fixture.